Networking/Model/NetworkingConfiguration.swift

```diff
@@ -24,12 +24,14 @@ private enum CommunicationProtocol: String {
  */
 public struct NetworkingConfiguration {
     
-    fileprivate let _useSecureConnection: Bool
-    fileprivate let _domainURL: String
-    fileprivate let _port: Int?
-    fileprivate let _subdomainURL: String?
-    
-    fileprivate let _usePinningCertificate: Bool
+    public var useSecureConnection: Bool = true
+    public var domainURL: String = ""
+    public var port: Int? = .none
+    public var subdomainURL: String? = .none
+    public var timeout: Double = 75.0
+    public var usePinningCertificate: Bool = false
+    public var pollTime: Double = 1.0
+    public var maximumPollingRetries: Int = 10
```

Comment:
Shouldn't this be an optional? Maybe you want to retry "eternally" or until the `pollTime` ends?
Also, `pollTime` is the time between one poll and the following one (`pollBreakTime` maybe?) or the time limit for the polling request in total (from the first poll to the last one)?
Please, now that this is public add documentation.

Reply:
`pollTime` is the time between one poll and the next. `pollBreakTime` seems confusing, maybe we could call it `secondsBetweenPolls` or something clearer.
There's no hard time limit to the polling, the timeout is determined by `maximumPollingRetries` * `pollTime`. I'm not sure if we want to retry eternally as the default behaviour (actually I don't think you would want to retry eternally in any case), maybe if the user does not set a `maximumPollingRetries` we could set it to `timeout`/`pollTime` (so the polling will last the same amount of time as any request before giving a `timeout`).

Reply:
I like that idea for the default value of `maximumPollingRetries`. And if we see that the "eternal" polling is something we need we can then change the none to default to forever.
About the naming, yeah, I like `secondsBetweenPolls`, it's clear.